dotnet/src/SemanticKernel.Abstractions/SemanticKernel.Abstractions.csproj

```
@@ -4,7 +4,7 @@
     <AssemblyName>Microsoft.SemanticKernel.Abstractions</AssemblyName>
     <RootNamespace>Microsoft.SemanticKernel</RootNamespace>
     <TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
-    <NoWarn>$(NoWarn);SKEXP0001;NU5104;SKEXP0120</NoWarn>
+    <NoWarn>$(NoWarn);NU5104;SKEXP0001;NU5104;SKEXP0120</NoWarn>
```

The exclude is already there.

```suggestion
    <NoWarn>$(NoWarn);SKEXP0001;NU5104;SKEXP0120</NoWarn>
```